content/docs/getting-started/installation/installationwizard/src/static/js/ui-functions.js

```diff
@@ -576,11 +576,11 @@ function displayCommands(releaseNameValue, commandTitleValue, commandNoteValue,
 		case "1.13.0":
 			helmChartVersion = CONSTANTS.CSM_HELM_V1130;
 			break;
-		case "1.14.0":
-			helmChartVersion = CONSTANTS.CSM_HELM_V1140;
+		case "1.14.1":
+			helmChartVersion = CONSTANTS.CSM_HELM_V1141;
```

Comment:
Could you check if the constant `CSM_HELM_V1141` is already defined? If it’s not there, please add it.